content/workstation/install.md

```diff
@@ -11,9 +10,14 @@ weight = 20
 
 {{< readfile file="/content/reusable/md/workstation_modularize.md" >}}
 
-## Supported platforms
+## System requirements
 
-Chef Workstation 26 RC3 and its components are supported on Linux x86-64 systems.
+Chef Workstation 26 RC3 has the following requirements:
+
+- Linux x86-64 (64-bit) systems only
+- Chef Habitat 1.6.0 or later installed
+- Minimum 2GB available disk space for installation
+- Internet connectivity for package downloads (or access to internal Habitat Builder)
 
 ## Prerequisites
```

Comment:
@IanMadd  Can we also include the habitat license key setup in the prerequisites? 

https://docs.chef.io/habitat/builder_profile/